src/harvest-rpde.js

```diff
@@ -228,4 +207,22 @@ async function harvestRPDE({
   }
 }
 
-module.exports = { harvestRPDE };
+/**
+ * @param {HarvestRpdeArgs} args
+ *
+ * @returns {Promise<void>} Only returns if there is a fatal error.
+ */
+function harvestRPDE(args) {
+  return baseHarvestRPDE(args, false);
+}
+
+/**
+ * @param {HarvestRpdeArgs} args
```

Comment:
Ah so the types aren't well defined yet 😅  but it's still useful that these are two functions because eventually you can turn this into `HarvestRpdeArgs<string>` where `HarvestRpdeArgs` has a `TModified extends string | number` template.

Might be worth putting something like that into a comment to deter anyone in future from merging them into one function